framework/config/src/test/java/org/apache/cloudstack/framework/config/ConfigKeyScheduledExecutionWrapperTest.java

```diff
@@ -59,8 +58,8 @@ public void nullConfigKeyTest() {
     @Test(expected = IllegalArgumentException.class)
     public void invalidConfigKeyTest() {
         TestRunnable runnable = new TestRunnable();
-        ConfigKey<String> configKey = Mockito.mock(ConfigKey.class);
-        when(configKey.value()).thenReturn("test");
+        ConfigKey<String> configKey = new ConfigKey<>(String.class, "test", "test", "test", "test", true,
+                ConfigKey.Scope.Global, null, null, null, null, null, ConfigKey.Kind.CSV, null);
```

Comment:
isn't using a mock fine?

Reply:
I think the original test (before Wei's change) was just testing for an invalid configKey; I reverted the change to keep author's original intent.